src/apicast-cli/cli/busted.lua

```diff
@@ -0,0 +1,16 @@
+local actions = require "apicast-cli.cmd"
+local colors = require "ansicolors"
+
+local function call(_, parser)
+  local busted_run = parser:command("b busted", "Runs busted tests.")
+    :action(actions.busted.run)
+
+  busted_run:option("-p", "only run test files matching the Lua pattern")
```

Comment:
it is, to defined all available options.. otherwise it says:

```
bin/apicast-cli b -f ddddasdasd -r 10

Usage: apicast-cli b [-h]

Error: unknown option '-f'
Did you mean '-h'?
```